runtime/compiler/p/codegen/J9PPCWatchedInstanceFieldSnippet.cpp

```diff
@@ -1,65 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2019, 2020 IBM Corp. and others
- *
- * This program and the accompanying materials are made available under
- * the terms of the Eclipse Public License 2.0 which accompanies this
- * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
- * or the Apache License, Version 2.0 which accompanies this distribution and
- * is available at https://www.apache.org/licenses/LICENSE-2.0.
- *
- * This Source Code may also be made available under the following
- * Secondary Licenses when the conditions for such availability set
- * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
- * General Public License, version 2 with the GNU Classpath
- * Exception [1] and GNU General Public License, version 2 with the
- * OpenJDK Assembly Exception [2].
- *
- * [1] https://www.gnu.org/software/classpath/license.html
- * [2] http://openjdk.java.net/legal/assembly-exception.html
- *
- * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
- *******************************************************************************/
-
-#include "codegen/J9PPCWatchedInstanceFieldSnippet.hpp"
-#include "p/codegen/PPCTableOfConstants.hpp" //PTOC
-
-
-uint8_t *TR::J9PPCWatchedInstanceFieldSnippet::emitSnippetBody()
-   {
-   // Call Base class's emitSnippetBody() to populate the snippet
-   // Here Binary Buffer will be ahead by the size of the struct J9JITWatchedInstanceFieldData
-   uint8_t *snippetLocation = (TR::J9WatchedInstanceFieldSnippet::emitSnippetBody() - TR::J9WatchedInstanceFieldSnippet::getLength(0));
```

Comment:
I suppose `TR::J9WatchedInstanceFieldSnippet::emitSnippetBody() ` is still going to happen?

Reply:
Yes. By removing the overridden version of `emitSnippetBody()` from this class, the superclass implementation from `TR::J9WatchedInstanceFieldSnippet` will be called directly.